src/bindings/python/wheel/guide-to-wheels.rst

```diff
@@ -0,0 +1,66 @@
+Python Wheels in OpenVINO™ Toolkit
+===================================
+
+Introduction
+#############
+Python wheels are a distribution format that allows for faster and more efficient installation of Python packages.
+In the context of OpenVINO™ Toolkit, Python wheels provide a convenient way to work with the toolkit in your Python projects.
+This section will guide you through the usage of Python wheels in OpenVINO™.
+
+What are Python Wheels?
+########################
+Python wheels are binary distribution formats that aim to make the installation of Python packages faster and more reliable.
+In the case of OpenVINO™, wheels are particularly useful for quick and hassle-free deployment. You can find OpenVINO™ Python distribution on PyPI: `OpenVINO on PyPI. <https://pypi.org/project/openvino/#files>`__
+
+Available Wheels in OpenVINO™
+++++++++++++++++++++++++++++++
+OpenVINO™ offers two main wheels:
+  
+`openvino <https://github.com/openvinotoolkit/openvino/tree/master/src/bindings/python/wheel/setup.py>`__: Core Python bindings for OpenVINO.
+
+`openvino-dev <https://github.com/openvinotoolkit/openvino/blob/master/tools/openvino_dev/setup.py>`__: Additional development tools and utilities.
+
+Building Python Wheels
+#######################
+To build Python wheels during the project build process, make sure to include the following options when running the cmake command:
+  1. bash
+  2. Copy code
+  3. cmake -DENABLE_PYTHON=ON -DENABLE_WHEEL=ON ..
+
+This ensures that the build process includes the Python components and generates the necessary wheel files.
+
+Locating Python Wheels
+#######################
+After building the project, you can find the generated Python wheels under the **openvino_install_dir/tools** directory.
```

Comment:
```suggestion
After building the project, you can find the generated Python wheels under the **openvino/build/wheels/** directory.
```
I think this is the easiest way to locate them after building the project and this way should be advised.